lib/matplotlib/backends/qt_compat.py

```diff
@@ -83,16 +83,16 @@
 if QT_API is None:
     # No ETS environment or incompatible so use rcParams.
     if rcParams['backend'] == 'Qt5Agg':
-        QT_API = rcParams['backend.qt5']
+        QT_API = QT_API_PYQT5
```

Comment:
OK, but this does't just deprecate the rcparam, it now ignores it, right?  Is that what you wanted?  Feel free to clear if so, but putting a red x so its noted..

Reply:
Yes, but the rc deprecation tools do not have a gentler option